diff --git a/node_modules/xlsx/xlsx.js b/node_modules/xlsx/xlsx.js
index d9b2278..1fbea8b 100644
--- a/node_modules/xlsx/xlsx.js
+++ b/node_modules/xlsx/xlsx.js
@@ -2888,7 +2888,7 @@ function split_regex(str, re, def) {
 function getdatastr(data) {
 	if(!data) return null;
 	if(data.data) return debom(data.data);
-	if(data.asNodeBuffer && has_buf) return debom(data.asNodeBuffer().toString('binary'));
+	if(data.asNodeBuffer && has_buf) return debom(data.asNodeBuffer().toString());
 	if(data.asBinary) return debom(data.asBinary());
 	if(data._data && data._data.getContent) return debom(cc2str(Array.prototype.slice.call(data._data.getContent(),0)));
 	if(data.content && data.type) return debom(cc2str(data.content));
